### DISCUSSION TITLE
Prevent modals from closing on escape key or clicking outside the modal

### DISCUSSION THREAD

**Discussion by appique-skynet** [filamentphp/filament#3089] [Help]
Is there a way to configure a modal in such a way that it won't close when pressing escape or clicking outside the modal?

Especially in the case of more complex forms in a modal, it can be a little annoying when accidentally clicking outside the modal and loosing all input.

**Comment by danharrin** [comment]
I think I agree that clicking outside the modal should not close it, but the escape key should still do that

PR the change and I will merge it

**Reply by hxnk** [reply]
PR'ed in #3097 (Same guy, different account 😉)
